RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Coordinator/RestoreProfileFromBackupCoordinator.swift

```diff
@@ -88,6 +89,7 @@ public struct RestoreProfileFromBackupCoordinator: Sendable, FeatureReducer {
 			state.profileSelection = .init(snapshot: profileSnapshot, isInCloud: isInCloud)
 			return .run { send in
 				try? await clock.sleep(for: .milliseconds(300))
+				try await radixConnectClient.connectToP2PLinks(profileSnapshot.appPreferences.p2pLinks)
```

Comment:
cant remember now, but what are we infact awaiting here? looks wrong? This mean that `send(.internal(.delay..(.importMnemonicsFlow))` event is not emitted until `radixConnectClient.connectToP2PLinks` has finished? So what happens if you quit Chrome and redo your demo in the video? Will this `RestoreProfileFromBackupCoordinator` reducer wait forever until connected? Or are we using unstructured concurrency inside `connectToP2PLinks` ensuring it will not be 'blocking"? Otherwise, perhaps this `try await` inside this `run` must change, e.g. to a `run {}.concat {}` or whatever the syntax is block, or possible even `Task` (unstructured concurrency)?

Reply:
> Will this RestoreProfileFromBackupCoordinator reducer wait forever until connected?

No, `connectToP2PLinks` calls in turn the `try await rtcClients.connect(password, waitsForConnectionToBeEstablished: false)`.